src/packages/@ncigdc/components/analysis/SelectScrnaSeq/SelectScrnaSeq.relay.js

```diff
@@ -52,11 +52,10 @@ export default (Component) =>
   )((props) => {
     return (
       <Query
-        parentProps={props}
-        name="SelectScrnaSeqCases"
-        minHeight={387}
-        variables={variables}
         Component={Component}
+        minHeight={387}
```

Comment:
What's this number for/from?

Reply:
it was originally the default height for card plots in clinical analysis, but it works ok here. it sets the height of the element before the query loads